testing/tests/DevExpress.ui.widgets.editors/tagBox.tests.js

```diff
@@ -5421,6 +5421,26 @@ QUnit.module("regression", {
         assert.equal($.trim($tagContainer.text()), "Item1Item3", "selected values are rendered");
     });
 
+    QUnit.test("selection should work with pregrouped data without paging and with preloaded datasource", (assert) => {
+        const ds = new DataSource({
+            store: [
+                { key: "Category 1", items: [{ id: 11, name: "Item 11" }, { id: 12, name: "Item 12" }] },
+                { key: "Category 2", items: [{ id: 21, name: "Item 21" }, { id: 22, name: "Item 22" }] }
+            ]
+        });
+        ds.load();
```

Comment:
It's very unobvious behavior. Can we avoid it?
Perhaps, we can turn off the `deferRendering` option for TagBox's Popup if editor has the initial value, so the List will immediately load the data from the DataSource